google-cloud-clients/google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/models/BulkMutation.java

```diff
@@ -34,7 +34,7 @@
  * <p>This class is meant for manual batching, while {@link BulkMutationBatcher} is meant for
  * automatic batching with flow control.
  */
-public final class BulkMutation implements Serializable {
+public final class BulkMutation implements Serializable, Cloneable {
```

Comment:
Please add a test to make sure that clone doesn't share data with the original and has all of the necessary attributes

Reply:
I have added a test case for this in `BulkMutation#cloneTests()`.